content/documentation/install-uninstall.md

```diff
@@ -6,61 +6,57 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
 ---
 
 {{< q-class "box warning" >}}
-- Quire is in a closed beta. Installation will not work unless you have [requested and been given access](https://goo.gl/forms/Ih4SOHje0Z9dm3C73).
+- Quire is in a limited beta, © J. Paul Getty Trust, and not yet released as open source software. For a free license to use Quire for your publication projects (and for additional access to support from Getty staff) [please sign up](https://goo.gl/forms/Ih4SOHje0Z9dm3C73).
```

Comment:
I like this, but I want to discuss this messaging a little more and what it means when we say signing up means "they have access to Getty staff."

Reply:
Agreed. This was just a starting point for sure. And in fact the bit about access to staff maybe isn't necessary here at all. Happy for any edits or suggestions. Maybe we can go over it on our Monday call.

Reply:
Sounds good! For now, maybe we could just remove the Getty staff part and leave it at "access to support" because that could include the forum. We can explain in more depth in the actual sign-up form. (I've made notes on how we might edit the form which we can go over on Monday!)